editions/tw5.com/tiddlers/community/resources/_Noteself_ by Danielo Rodríguez.tid

```diff
@@ -0,0 +1,29 @@
+created: 20141122093837330
+modified: 20161231234157548
+tags: Resources
+title: "Noteself" by Danielo Rodríguez
+type: text/vnd.tiddlywiki
+url: https://noteself.github.io/
+
+~NoteSelf is your personal, private, customizable, Evernote-like experience.
+You want cloud ? Fine! You don't ? Fine too! It's all yours, It's your decision !
```

Comment:
imo should be: `You want cloud? Fine! You don' ? Fine too! It's all yours, It's your decision!`